README.md

```diff
@@ -241,6 +241,24 @@ This SDK exposes the following classes:
 - `TransactionsWidget`
 - `TrendsWidget`
 
+### Troubleshooting
+
+#### Post messages not working
+
+Check the following items if widget post message callbacks are not being
+triggered in your application.
+
+- Ensure your SSO request is correctly configured for the Web SDK:
+    - `ui_message_version` should be set to `4`.
+    - `is_mobile_webview` should be set to `false`.
+- Ensure you are using the corresponding widget class for the `widget_type`
+  used in the SSO request. For example, if you set `widget_type` to
+  `connect_widget`, then you should use the `ConnectWidget` class.
+- Ensure you are serving your application's HTML from a web server and not
+  loading it from your file system.
+- Ensure your application's [`Referrer-Policy`][referrer_policy] is one that
```

Comment:
Do we know which `Referrer-Policy` values do and do not work? I'm guessing any policy that sends us the Referrer header will work. So same-origin won't work, but most others will (maybe all others), including [`strict-origin`](https://www.w3.org/TR/referrer-policy/#referrer-policy-strict-origin) depending on how they set it up -- it'd need to test though.

Also, I know `target_origin_referrer` is new and not documented, but I think we're gonna need something like the paragraph below in the future:

> If the policy you are using is not allowing origin information to be passed to our widget, then you may use `target_origin_referrer` to specify your application's domain in the SSO request. For example, if your domain is `https://www.mx.com/`, then you should include `target_origin_referrer: "https://www.mx.com/"` in your SSO request.

Reply:
I think we should only recommend the default and leave it at that for now: `strict-origin-when-cross-origin`

Reply:
@sambev are you saying we should make the recommendation? How about this:

```markdown
- Ensure your application's [`Referrer-Policy`][referrer_policy] is one that
  allows origin information to be passed to the widget. We recommend using
  the default, which is `strict-origin-when-cross-origin`
```

Reply:
:shipit:

Reply:
@sambev done and done.